lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -154,7 +154,7 @@ struct ConvertLayoutOpConversion
     }
     if (auto mmaLayout = layout.dyn_cast<MmaEncodingAttr>()) {
       auto shapePerCTA = getShapePerCTA(mmaLayout, shape);
-      auto instrShape = mmaLayout.getInstrShape();
+      auto instrShape = mmaVersionToInstrShape(mmaLayout, shapePerCTA);
```

Comment:
Can we remove inputType from mmaEncoding if we change the function signature to `mmaVersionToInstrShape(Type);`. And other functions in the same way, if need.